### PR TITLE
Catch SIGTERM and SIGINT

### PR DESCRIPTION
**Pull request by jlevesy**
Deals with https://github.com/upfluence/man/issues/4

### CODE REVIEW

**Comment by jlevesy** [comment]
@AlexisMontagne RTM ?

File: etcdenv/context.go [hunk 1]
```diff
@@ -176,22 +176,24 @@ func (ctx *Context) Run() {
...
 			ctx.Runner.Restart(ctx.CurrentEnv)
+			log.Println("Process restarted")
 		case <-ctx.ExitChan:
+			log.Println("Asking the runner to stop")
 			ctx.Runner.Stop()
+			log.Println("Runner stopped")
 		case status := <-processExitChan:
-			log.Println(fmt.Sprintf("Process exited with the status %d", status))
-
+			fmt.Printf("Child process exited with status %d\n", status)
```
Comment (AlexisMontagne) [inline etcdenv/context.go]: not  `fmt.Printf` but `log.Printf`

**Comment by jlevesy** [comment]
@AlexisMontagne  RTM ?

**Comment by AlexisMontagne** [comment]
LGTM

**Pull request merged**